### PR TITLE
Add region change feature

### PR DESCRIPTION
**Pull request by broccoder**
Adds the region change feature that was requested in #11

### CODE REVIEW

File: MineColab.ipynb [hunk 1]
```diff
@@ -42,7 +42,7 @@
         "\n",
         "---\n",
         "\n",
-        "The script below will run your server. You'll have to create a server first to be able to use it - don't worry, the scripts below will do the majority of the work for you."
+        "The script below will run your server. You'll have to create a server first to be able to use it - don't worry, the scripts below will do the majority of the work for you. You might also want to change the default region to your region, check below."
```
Comment (infinitygamer404) [inline MineColab.ipynb:45]: @thecoder-001 you might wanna modify this one because I'm not sure what I should put here to tell people to change the region.

**Comment by infinitygamer404** [comment]
Sorry for hastily dropping this. I was using this reference btw: https://pyngrok.readthedocs.io/en/latest/#setting-the-region
image

File: MineColab.ipynb (hunk 1 above)
Comment (thecoder-001) [reply MineColab.ipynb:45]: yup, will do

**Comment by thecoder-001** [comment]
Looks good.
Thanks for contributing!

**Comment by infinitygamer404** [comment]
Ah dang I missed a comma right there
image
Sorry
Hopefully next time I'm gonna be better

**Comment by jasnoor09** [comment]
plz help me where do i have to paste this command

**Pull request merged**